ci.nix

```diff
@@ -72,6 +72,7 @@ mkShell {
   SPDK_PATH = if nospdk then null else "${spdk}";
   FIO_SPDK = if nospdk then null else "${spdk}/fio/spdk_nvme";
   ETCD_BIN = "${etcd}/bin/etcd";
+  LVM_BINS = "${lvm2.bin}/bin";
```

Comment:
Probably we have to allow to disable LVM.

Reply:
This doesn't control lvm enablement, only brings the binaries to the shell.